code/drasil-data/Data/Drasil/Equations/Defining/Physics.hs

```diff
@@ -20,16 +20,16 @@ newtonSLRel, weightEqn, weightDerivAccelEqn, weightDerivNewtonEqn, weightDerivRe
   weightDerivSpecWeightEqn,
   hsPressureEqn, accelerationEqn, velocityEqn  :: Relation
 
-newtonSLRel               = sy QP.force $= sy QPP.mass * sy QP.acceleration
+newtonSLRel               = sy QP.force $= mulRe (sy QPP.mass) (sy QP.acceleration)
 
-weightEqn                 = sy QP.weight $= sy QPP.vol * sy QPP.specWeight
-weightDerivNewtonEqn      = sy QP.weight $= sy QPP.mass * sy QP.gravitationalAccel
-weightDerivReplaceMassEqn = sy QP.weight $= sy QPP.density * sy QPP.vol * sy QP.gravitationalAccel
-weightDerivSpecWeightEqn  = sy QP.weight $= sy QPP.vol * sy QPP.specWeight
+weightEqn                 = sy QP.weight $= mulRe (sy QPP.vol) (sy QPP.specWeight)
+weightDerivNewtonEqn      = sy QP.weight $= mulRe (sy QPP.mass) (sy QP.gravitationalAccel)
+weightDerivReplaceMassEqn = sy QP.weight $= mulRe (sy QPP.density) (mulRe (sy QPP.vol) (sy QP.gravitationalAccel))
+weightDerivSpecWeightEqn  = sy QP.weight $= mulRe (sy QPP.vol) (sy QPP.specWeight)
 
-weightDerivAccelEqn       = sy QP.acceleration $= vec2D 0 (sy QP.gravitationalAccel * sy QM.unitVectj)
+weightDerivAccelEqn       = sy QP.acceleration $= vec2D (int 0) (mulRe (sy QP.gravitationalAccel) (sy QM.unitVectj))
```

Comment:
This wouldn't type... the first part is an integer, the 2nd part a real.

Reply:
Thank you, this should be fixed now